src/ui/public/courier/saved_object/saved_object.js

```diff
@@ -259,12 +259,47 @@ export default function SavedObjectFactory(es, kbnIndex, Promise, Private, Notif
     }
 
     /**
+     * Attempts to create the current object using the serialized source. If an object already
+     * exists, a warning message requests an overwrite confirmation.
+     * @param source - serialized version of this object (return value from this.serialize())
+     * What will be indexed into elasticsearch.
+     * @returns {Promise} - A promise that is resolved with the objects id if the object is
+     * successfully indexed. If the overwrite confirmation was rejected, an error is thrown with
+     * a confirmRejected = true parameter so that case can be handled differently than
+     * a create or index error.
+     * @resolved {String} - The id of the doc
+     */
+    const createSource = (source) => {
+      return docSource.doCreate(source)
+        .catch((err) => {
+          // record exists, confirm overwriting
+          if (_.get(err, 'origError.status') === 409) {
+            const confirmMessage = 'Are you sure you want to overwrite ' + this.title + '?';
```

Comment:
Huh, worse than the message being wrong, it will actually warn me it's going to override a document, and then it doesn't. Is the import still matching on the title instead of the id?